src/NewThumb/NewThumb.js

```diff
@@ -231,6 +231,14 @@ module.exports = kind(
 	},
 
 	/**
+	* Override `handleResize()` to re-calculate ratio and size.
+	* @private
+	*/
+	handleResize: function () {
+		this.calculateMetrics();
```

Comment:
Generally speaking, yes it is always better to call super if there it is. :)
But, when I looked up the original behavior of UIComponent.handleResize, it is calling reflow of layout.
I thought it will be better to short circuit in this particular case because thumb is not using any layout kind.